content/attacks/posts/2022-01-17-Crypto.com.md

```diff
@@ -0,0 +1,29 @@
+---
+date: 2022-01-17
+custodians: Crypto.com
+categories: Custodian
+title: "Crypto.com Confirms $34M Hack"
+---
+
+## Summary
+
+An attacker found a way to approve transactions without the two-factor authentication control being inputted by account holders. This resulted in 483 Crypto.com users losing funds from their accounts. The exchange reaffirmed comments made by the firm’s CEO, Kris Marszalek, that any accounts found to be impacted were fully restored, resulting in no loss of funds for users.  
+In response to the incident, Crypto.com has added an additional layer of security to withdrawals. Users will now need to wait 24 hours after registering a new withdrawal address before transferring funds to it. The exchange also says it has engaged with third-party security firms to perform additional security checks.
+Also Crypto.com has launched a new Worldwide Account Protection Program. The program promises to restore funds up to $250,000 for qualified users in the event of fraud or theft. To qualify, users must meet a series of criteria, such as having two-factor authentication enabled on all transactions and filing a report with local police.
```

Comment:
Rewrite the Summary section with your own words, mentioning what Crypto.Com is and when it happened. And then, what actually happened with factual explanations. Otherwise, it looks like it is the Summary written by Crypto.Com